doc/source/user_guide/virtual_env_motorcad.rst

```diff
@@ -0,0 +1,61 @@
+.. _ref_virtual_env_motorcad:
+
+Using Python Virtual Environment in Motor-CAD
+===============================
+
+User have a flexibility to use python virtual environment with Motor-CAD. By default Motor-CAD uses python 
+comes with Motor-CAD installation. 
+
+.. figure:: ../images/defaults_python_path.PNG
+    :width: 500pt
+
+Creating a new local Python virtual environment
```

Comment:
```suggestion
Create a local Python virtual environment
```
Use simple verbs (with -ing ending) and no need for "new" as you cannot create an old one.